### DISCUSSION TITLE
The reload accurs too offen

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1254] [Q&A] [answered]
Reloading happens right away when we change the code,even if we only enter the letter a, This resulted in many compilation errors。 The reload should happen when we press the"CTRL+S" and save the file.

**Comment by rodja** [comment, marked as answer]
Reload uses uvicorn which itself uses watchfiles... and this package really only watches for file changes. Maybe you have some kind of autosave active? Could you try another editor?

**Reply by v479038280** [reply]
Maybe which plugin changed my settings, that's the reason
